dev-docs/howtoguides/run_dep8_tests.md

```diff
@@ -0,0 +1,26 @@
+# How to run dep8 tests
+
+Our package currently supports [dep8
+tests]( https://salsa.debian.org/ci-team/autopkgtest/-/blob/master/doc/README.package-tests.rst).
+Those tests can be found under `debian/tests/usage` and although the test is running
+some Pro client commands, the idea is to verify if any modification to our package dependencies
```

Comment:
This is a bit unclear - it's simultaneously referring to multiple tests and one test, and "some commands" but also "this command". Could we make this more specific?